packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/utils/getListInfo.ts

```diff
@@ -99,31 +106,41 @@ const bulletListType: Record<string, number> = {
     '>': BulletListType.ShortArrow,
 };
 
-const identifyNumberingListType = (textBeforeCursor: string): NumberingListType => {
-    const numbering = textBeforeCursor.replace(/\s/g, '');
-    const char = identifyCharacter(numbering);
-    const numberingType = identifyNumberingType(numbering);
-    return numberingListTypes[numberingType](char);
+const identifyNumberingListType = (numbering: string): NumberingListType | null => {
+    const number = numbering.length === 2 ? numbering[0] : numbering[1];
+    const separator = numbering.length === 2 ? numbering[1] : numbering[0];
+    const char = identifyCharacter(separator);
+    const numberingType = identifyNumberingType(number);
+    return char && numberingType ? numberingListTypes[numberingType](char) : null;
 };
 
-const identifyBulletListType = (textBeforeCursor: string): BulletListType => {
-    const bullet = textBeforeCursor.replace(/\s/g, '');
-    return bulletListType[bullet];
+const identifyBulletListType = (bullet: string): BulletListType | null => {
+    return bulletListType[bullet] || null;
 };
 
 /**
  * @internal
  * @param textBeforeCursor The trigger character
  * @param listType The type of the list (ordered or unordered)
- * @returns the style of the list
+ * @returns The info with type and style of the list
  */
-export default function getListStyle(
-    textBeforeCursor: string,
-    listType: ListType
-): NumberingListType | BulletListType {
-    if (listType === ListType.Ordered) {
-        return identifyNumberingListType(textBeforeCursor);
+export default function getListInfo(textBeforeCursor: string): ListInfo {
+    const trigger = textBeforeCursor.replace(/\s/g, '');
```

Comment:
This will replace all spaces, including the spacing between letters, is that expected? If you only want to remove leading and trailing spaces, you can use .trim()